pyproject.toml

```diff
@@ -12,31 +12,30 @@ classifiers = [
     "Environment :: Plugins",
     "Framework :: Django :: 4.2",
     "Framework :: Django :: 5.0",
+    "Framework :: Django :: 5.1",
+    "Framework :: Django :: 5.2",
     "Framework :: Django",
     "Intended Audience :: Developers",
     "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3 :: Only",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 packages = [{include = "easyaudit"}]
 
 [tool.poetry.dependencies]
-python = "^3.8"
-django = ">=4.2,<6.0"
+python = "^3.9"
+django = "4.2.20"
```

Comment:
Curious about this change. Maybe I'm missing something but why are we limiting django to just one version?

Reply:
Sorry, my bad.